packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableClaimableRow.tsx

```diff
@@ -28,18 +28,20 @@ import { formatAmount } from '../../../util/NumberUtils'
 import { sanitizeTokenSymbol } from '../../../util/TokenUtils'
 import { TransactionsTableRowAction } from './TransactionsTableRowAction'
 import { useRemainingTime } from '../../../state/cctpState'
+import { useChainLayers } from '../../../hooks/useChainLayers'
 
 type CommonProps = {
   tx: MergedTransaction
   isSourceChainArbitrum: boolean
 }
 
 function ClaimableRowStatus({ tx, isSourceChainArbitrum }: CommonProps) {
+  const { parentLayer, layer } = useChainLayers()
   const matchingL1Tx = tx.isCctp
     ? tx.cctpData?.receiveMessageTransactionHash
     : findMatchingL1TxForWithdrawal(tx)
-  const fromNetwork = isSourceChainArbitrum ? 'L2' : 'L1'
-  const toNetwork = isSourceChainArbitrum ? 'L1' : 'L2'
+  const fromNetwork = isSourceChainArbitrum ? layer : parentLayer
```

Comment:
What if we returned `fromLayer` and `toLayer` to indicate the direction?

So the it would look something like:

`const { parentLayer, layer, fromLayer, toLayer } = useChainLayers()`

Reply:
After looking more into it, we don't even need to make these directional checks anymore. So that we can use `parentLayer` and `layer` directly, and we don't have to further modify the hook.

The hook can stay as:
```
const { parentLayer, layer } = useChainLayers()
```